app/src/main/java/com/keylesspalace/tusky/TabPreferenceActivity.kt

```diff
@@ -317,6 +317,7 @@ class TabPreferenceActivity : BaseActivity(), Injectable, ItemInteractionListene
                     showProgressBarJob.cancel()
                     adapter.addAll(lists)
                     if (lists.isEmpty()) {
+                        // TODO do like in ListsForAccountFragment (full binding in dialog with message view)
```

Comment:
Mostly the draft is about that I have the feeling that all list lists should be joined more or less.

For example the list selection in the tab preferences is basically the same as here (dialog which loads a list list and you can tap an entry). Also visually all list lists should look the same.

So a general lists fragment (?) could be used in at least 3 locations (third being the ListActivity itself).

Reply:
Yes consolidating all those similar things would be really cool

Reply:
(but also see "architectually clean" question below)